controller/cache/cache.go

```diff
@@ -69,6 +70,10 @@ const (
 
 	// EnvClusterCacheRetryUseBackoff is the env variable to control whether to use a backoff strategy with the retry during cluster cache sync
 	EnvClusterCacheRetryUseBackoff = "ARGOCD_CLUSTER_CACHE_RETRY_USE_BACKOFF"
+
+	// AnnotationIgnoreResourcesUpdate when set to true on an untracked resource,
+	// argo will apply `ignoreResourceUpdates` configuration on it.
+	AnnotationIgnoreResourcesUpdate = "argocd.argoproj.io/ignore-resources-update"
```

Comment:
Let's make the name consistent. Make sure to validate all references

```suggestion
	AnnotationIgnoreResourceUpdates = "argocd.argoproj.io/ignore-resource-updates"
```